camera_parameters:
  ros__parameters:
    camera_matrix:
      cols: 3
      data:
      - 914.361813321524
      - 0
      - 467.01160766470457
      - 0
      - 913.3059607363589
      - 349.29006094091017
      - 0
      - 0
      - 1
      rows: 3
    distortion_coefficients:
      cols: 5
      data:
      - -0.006536274709474736
      - -0.07577132181064014
      - -0.0009035700680331949
      - -0.0010633175307964142
      - 0.3442948926409829
      rows: 1
